tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "target": "es6",                                
+    "module": "es2015",                          
+    "outDir": "./Public",                       
+    "rootDir": "./Src",                            
```

Comment:
nitpick: rename `Src/` to `src` as most naming conventions on the FE are lowercase.